### PR TITLE
Bump terraform version to 1.1.7

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This change to .tf files, `pkg/terraform/terraform.go`, `pkg/terraform/exec/exec.go` are as per the latest Terraform.

### CODE REVIEW

**Comment by Rajalakshmi-Girish** [comment]
> @Rajalakshmi-Girish: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> pull-kubetest2-plugins-terraform-validate 	[d98471a](https://github.com/ppc64le-cloud/kubetest2-plugins/commit/d98471a9f3fc30bbe5c89f563b4a33d88b452d56) 	[link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_kubetest2-plugins/52/pull-kubetest2-plugins-terraform-validate/1501141848851222528) 	true 	`/test pull-kubetest2-plugins-terraform-validate`
> 
> [Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=kubetest2-plugins&pr=52). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:Rajalakshmi-Girish).

@mkumatag https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/kubetest2-plugins/kubetest2-plugins-presubmit.yaml has to be modified to latest.

**Comment by mkumatag** [comment]
> @mkumatag: GitHub didn't allow me to request PR reviews from the following users: Prajyot-Parab, please, review, this, PR.
> 
> Note that only [ppc64le-cloud members](https://github.com/orgs/ppc64le-cloud/people) and repo collaborators can review this PR, and authors cannot review their own PRs.

invitation sent @Prajyot-Parab, ptal

**Comment by mkumatag** [comment]
> @mkumatag https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/kubetest2-plugins/kubetest2-plugins-presubmit.yaml has to be modified to latest.

lets modify then :)

**Comment by Prajyot-Parab** [comment]
> > @mkumatag: GitHub didn't allow me to request PR reviews from the following users: Prajyot-Parab, please, review, this, PR.
> > Note that only [ppc64le-cloud members](https://github.com/orgs/ppc64le-cloud/people) and repo collaborators can review this PR, and authors cannot review their own PRs.
> 
> invitation sent @Prajyot-Parab, ptal

Accepted and Joined.

**Comment by Rajalakshmi-Girish** [comment]
@mkumatag @Prajyot-Parab https://prow.ppc64le-cloud.org/view/s3/prow-logs/logs/tf-periodic-kubernetes-containerd-conformance-test-ppc64le/1501114788405579776 is a successful run with these changes. (Just that it had a few extra print statements)

**Comment by mkumatag** [comment]
> > @Rajalakshmi-Girish: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> > Test name 	Commit 	Details 	Required 	Rerun command
> > pull-kubetest2-plugins-terraform-validate 	[d98471a](https://github.com/ppc64le-cloud/kubetest2-plugins/commit/d98471a9f3fc30bbe5c89f563b4a33d88b452d56) 	[link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_kubetest2-plugins/52/pull-kubetest2-plugins-terraform-validate/1501141848851222528) 	true 	`/test pull-kubetest2-plugins-terraform-validate`
> > [Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=kubetest2-plugins&pr=52). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:Rajalakshmi-Girish).
> 
> @mkumatag https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/kubetest2-plugins/kubetest2-plugins-presubmit.yaml has to be modified to latest.

will merge once we fix the this job.

**Comment by Rajalakshmi-Girish** [comment]
> will merge once we fix the this job.

@mkumatag Can you PTAL https://github.com/ppc64le-cloud/test-infra/pull/287?

File: data/data/powervs/provider.tf [hunk 1]
```diff
@@ -1,5 +1,4 @@
 provider "ibm" {
-  version = ">= 1.9.0"
```
Comment (mkumatag) [inline data/data/powervs/provider.tf:2]: is this change needed? if yes - I don't see vfsdata is updated with this change! can you run generate code again to get included in the vfsdata?

Comment (Rajalakshmi-Girish) [reply data/data/powervs/provider.tf:2]: Yes, this is deprecated. Warning is thrown if we have version mentioned.
Hmm yes, I missed generating the code. Will submit a patch.

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve
